diff --git a/node_modules/react-native-android-notification-listener/android/build.gradle b/node_modules/react-native-android-notification-listener/android/build.gradle
index 1274a12..22aa25b 100644
--- a/node_modules/react-native-android-notification-listener/android/build.gradle
+++ b/node_modules/react-native-android-notification-listener/android/build.gradle
@@ -1,17 +1,3 @@
-buildscript {
-
-
-  repositories {
-      google()
-      mavenCentral()
-  }
-
-  dependencies {
-    classpath 'com.android.tools.build:gradle:4.2.1'
-    classpath 'com.google.code.gson:gson:2.8.6'
-  }
-}
-
 apply plugin: 'com.android.library'
 
 
@@ -24,6 +10,7 @@ def getExtOrIntegerDefault(name) {
 }
 
 android {
+  namespace "com.lesimoes.androidnotificationlistener"
 
   compileSdkVersion getExtOrIntegerDefault('compileSdkVersion')
 
@@ -40,8 +27,8 @@ android {
       minifyEnabled false
     }
   }
-  lintOptions {
-    disable 'GradleCompatible'
+  lint {
+    disable += 'GradleCompatible'
   }
   compileOptions {
     sourceCompatibility JavaVersion.VERSION_1_8
@@ -50,80 +37,16 @@ android {
 }
 
 repositories {
-  mavenCentral()
-  google()
-
-  def found = false
-  def defaultDir = null
-  def androidSourcesName = 'React Native sources'
-
-  if (rootProject.ext.has('reactNativeAndroidRoot')) {
-    defaultDir = rootProject.ext.get('reactNativeAndroidRoot')
-  } else {
-    defaultDir = new File(
-      projectDir,
-      '/../../../node_modules/react-native/android'
-    )
-  }
-
-  if (defaultDir.exists()) {
-    maven {
-      url defaultDir.toString()
-      name androidSourcesName
-    }
-
-    logger.info(":${project.name}:reactNativeAndroidRoot ${defaultDir.canonicalPath}")
-    found = true
-  } else {
-    def parentDir = rootProject.projectDir
-
-    1.upto(5, {
-      if (found) return true
-      parentDir = parentDir.parentFile
-
-      def androidSourcesDir = new File(
-        parentDir,
-        'node_modules/react-native'
-      )
-
-      def androidPrebuiltBinaryDir = new File(
-        parentDir,
-        'node_modules/react-native/android'
-      )
-
-      if (androidPrebuiltBinaryDir.exists()) {
-        maven {
-          url androidPrebuiltBinaryDir.toString()
-          name androidSourcesName
-        }
-
-        logger.info(":${project.name}:reactNativeAndroidRoot ${androidPrebuiltBinaryDir.canonicalPath}")
-        found = true
-      } else if (androidSourcesDir.exists()) {
-        maven {
-          url androidSourcesDir.toString()
-          name androidSourcesName
-        }
-
-        logger.info(":${project.name}:reactNativeAndroidRoot ${androidSourcesDir.canonicalPath}")
-        found = true
-      }
-    })
-  }
-
-  if (!found) {
-    throw new GradleException(
-      "${project.name}: unable to locate React Native android sources. " +
-      "Ensure you have you installed React Native as a dependency in your project and try again."
-    )
-  }
+    google()
+    mavenCentral()
 }
 
 
 
+
 dependencies {
     //noinspection GradleDynamicVersion
-  implementation "com.facebook.react:react-native:+"
+  implementation "com.facebook.react:react-android"
 // From node_modules
     implementation 'com.google.code.gson:gson:2.8.6'
 }
